litmus-portal/graphql-server/pkg/project/create.go

```diff
@@ -164,3 +163,39 @@ func getInvitation(ctx context.Context, member model.MemberInput) (dbSchema.Invi
 
 	return "", nil
 }
+
+//RemoveMember ...
+func RemoveMember(ctx context.Context, member model.MemberInput) (string, error) {
+	invitation, err := getInvitation(ctx, member)
+	if err != nil {
+		return "Unsuccessful", err
+	}
+	if invitation == dbSchema.AcceptedInvitation {
+		err = database.RemoveMember(ctx, member.ProjectID, member.UserName)
+		if err != nil {
+			return "Unsuccessful", err
+		}
+	} else {
+		return "Unsuccessful", errors.New("User is not a member of the project")
+	}
+	return "Member successfully removed", nil
+}
+
+//CancelInvite ...
+func CancelInvite(ctx context.Context, member model.MemberInput) (string, error) {
```

Comment:
We can merge these two functions also as for `Accepted` `removeMember` will be applicable and for `Pending` `cancelInvite will be applicable